content/pages/docs-as-code.html

```diff
@@ -129,129 +121,128 @@ <h1 class="ui huge header">
 
 <section>
   <div class="ui very padded container">
-    <div class="ui vertically padded centered grid" id="analytics">
+    <div class="ui vertically padded centered grid" id="{{ header_1 }}">
 
       {% call about.step(
           header=header_1,
-          image="/images/docs-as-code/analytics.png",
-          image_alt="Graph showing traffic and search analytics in a period",
+          image="/images/docs-as-code/flyout-zoomed.png",
+          image_alt="Flyout showing versions of a documentation project",
           icon=icon_1) %}
         {% markdown %}
-      Explore traffic analytics and understand what are the pages your readers read the most.
-      **Prioritize the work** to improve them before other pages with less traffic.
-      Find out what are the terms your users search the most and write documentation pages
-      that include those terms so they can easily find them.
+        Read the Docs can host multiple versions of your docs.
+        Keep your 1.0 and 2.0 documentation online,
+        so your yours can always find the right version.
         {% endmarkdown %}
+        <a class="ui basic primary button" href="https://docs.readthedocs.io/en/stable/versions.html">
```

Comment:
We have been using the page redirect for links in other places.

```suggestion
        <a class="ui basic primary button" href="https://docs.readthedocs.io/page/versions.html">
```